pkg/sdn/plugin/node.go

```diff
@@ -248,12 +252,9 @@ func (node *OsdnNode) Start() error {
 			return true, nil
 		})
 
-	log.V(5).Infof("Creating and initializing openshift-sdn pod manager")
-	node.podManager, err = newPodManager(node.host, node.localSubnetCIDR, node.networkInfo, node.kClient, node.policy, node.mtu, node.oc)
-	if err != nil {
-		return err
-	}
-	if err := node.podManager.Start(cniserver.CNIServerSocketPath); err != nil {
+	// Kubelet has initialized, now we have a valid node.host
```

Comment:
Erm... I *think* you're right, but given the fun we've had with startup ordering in the past (and that we're continuing to have now) I'd feel a little bit sketchy merging that to release-1.5 without it getting any testing in master first. How about we make that change for master but for release-1.5 leave it where it is and just clarify in the comment that everything *except* .host and .kubeletCniPlugin are initialized by that point.